### PR TITLE
MEGA.nz | Fixed the weird out of place gray border when in List view,…

### PR DESCRIPTION
**Pull request by stankovictab**
… changed the search bar color and removed an ugly fixed color for an element

old

new

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -7638,8 +7638,11 @@ i.transfer-progress-icon
 span.top-icon.language
 
 CSS
-.files-grid-view.fm .column-settings {
-    background-color: ${#181a1b} !important;
+.grid-wrapper {
+    outline: 0px !important;
```
Comment (ZOASR) [inline src/config/dynamic-theme-fixes.config:7642]: Maybe changing the border color is bettter than removing it completely!
```suggestion
    outline-color: var(--darkreader-bg--surface-main);
```

**Review by ZOASR — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:7642]: I agree with this one.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (stankovictab) [reply src/config/dynamic-theme-fixes.config:7642]: Yup, even better!

Comment (ZOASR) [reply src/config/dynamic-theme-fixes.config]: Great!, Thanks.

**Review by Gusted — APPROVED** [review]
Thank you!

**Pull request merged**